assets/js/checkout.js

```diff
@@ -184,109 +190,87 @@ $(function () {
       $('#addon-text-refarch').text('Reference Architecture');
     }
 
-    if (checkoutOptions.pro_support && checkoutOptions.setup_deployment) {
+    if (support && refarch) {
       $('.grunty-sprite').attr('data-sprite', 3);
     }
-
-    _calculatePrice();
   }
 
-  function _calculatePrice() {
-    var total, subtotal, subscriptionTotal;
+  function _updatePrice() {
+    var monthlyTotal, dueNowTotal;
 
     switch (checkoutOptions.subscription_type) {
       case 'aws':
-        total = subtotal = pricing.subscriptions.aws.price.value;
+        monthlyTotal = pricing.subscriptions.aws.price.value;
         break;
       case 'gcp':
-        total = subtotal = pricing.subscriptions.gcp.price.value;
+        monthlyTotal = pricing.subscriptions.gcp.price.value;
         break;
       default: // do nothing
     }
 
     if (checkoutOptions.pro_support) {
       switch (checkoutOptions.subscription_type) {
         case 'aws':
-          total += subtotal = pricing.subscriptions.aws.pro_support_price.value;
+          monthlyTotal += pricing.subscriptions.aws.pro_support_price.value;
           break;
         case 'gcp':
-          total += subtotal = pricing.subscriptions.gcp.pro_support_price.value;
+          monthlyTotal += pricing.subscriptions.gcp.pro_support_price.value;
           break;
         default: // do nothing
       }
     }
 
+    // CIS Compliance is only on AWS for now
+    if (checkoutOptions.setup_compliance) {
+      monthlyTotal += pricing.subscriptions.aws.cis_compliance_price.value;
+    }
+
+    dueNowTotal = monthlyTotal;
+
     // Only AWS supports the Ref Arch
     if (checkoutOptions.setup_deployment) {
```

Comment:
It's a little confusing because `checkoutOptions` still retains the older naming: `setup_deployment` vs `refarch` as you've named it here. Your naming is more up to date. Probably doesn't make sense to pay down the tech debt in this PR though..

Reply:
Yeah, agreed. We can clean up the tech debt when we switch over to a custom Recurly.js page. In the meantime, the shorthand local variables offer a decent and consistent way to map them.